shared/src/main/java/uk/gov/di/authentication/shared/services/ConfigurationService.java

```diff
@@ -648,4 +648,15 @@ public String getNotifyTemplateId(String templateName) {
     public URI getAccountInterventionServiceURI() {
         return URI.create(System.getenv("ACCOUNT_INTERVENTION_SERVICE_URI"));
     }
+
+    public boolean abortOnAccountInterventionsErrorResponse() {
+        return System.getenv()
+                .getOrDefault("ACCOUNT_INTERVENTION_SERVICE_ABORT_ON_ERROR", "false")
+                .equals("true");
+    }
+
+    public long getAccountInterventionServiceCallTimeout() {
+        return Long.parseLong(
+                System.getenv().getOrDefault("ACCOUNT_INTERVENTION_SERVICE_CALL_TIMEOUT", "5000"));
```

Comment:
The Auth implementation may be different (in which case ignore me) but Orch were initially suggested to use 5000ms but that was recently dropped to 1000ms